README.md

```diff
@@ -25,12 +25,28 @@ Check out our [roadmap](ROADMAP.md) as we push towards a __0.1.0__ release.
 ## Using Kedge
 
 ### Installation
-You can download latest master build for all major operating systems and architectures.
-- [Linux (amd64)][Bintray Latest Linux]
-- [macOS (darwin)][Bintray Latest macOS]
-- [Windows (amd64)][Bintray Latest Windows]
+You can download latest binary (built on each master PR merge) for [Linux (amd64)][Bintray Latest Linux], [macOS (darwin)][Bintray Latest macOS] or [Windows (amd64)][Bintray Latest Windows] from [Bintray](https://bintray.com):
 
-Or you can build it yourself from the master GitHub branch:
+```sh
+# Linux 
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64 -o kedge
+
+# macOS
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kedge-darwin-amd64 -o kedge
+
+# Windows
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kedge-windows-amd64.exe -o kedge
+
+# Linux + macOS
+chmod +x kedge
+sudo mv ./kedge /usr/local/bin/kedge
+
+# Windows
+# Copy kedge to your %GOPATH/bin directory
+move kedge %GOPATH%/bin/kedge
```

Comment:
For windows, it might be better to have just plain English sentence explaining what to do, and leave to rest on the user.

similarly to whats in minikube or kubectl :

Download the [kedge-windows-amd64.exe][Bintray Latest Windows]  file, rename it to kedge.exe and add it to your PATH.